network-api/networkapi/homepage/tests/tests_factory.py

```diff
@@ -0,0 +1,161 @@
+from django.test import TestCase
```

Comment:
this might just be a naming thing but are we actually testing the factory or are we testing the model here?

Reply:
@gideonthomas The idea here is that if we make changes to the model that cause the Factory to error out building model instances, we'd detect it.

Reply:
ok, i'm not picky about it tbh